orchagent/dash/dashrouteorch.cpp

```diff
@@ -76,11 +76,18 @@ bool DashRouteOrch::addOutboundRouting(const string& key, OutboundRoutingBulkCon
         SWSS_LOG_INFO("Retry as route group %s not found", ctxt.route_group.c_str());
         return false;
     }
-    if (ctxt.metadata.has_vnet() && gVnetNameToId.find(ctxt.metadata.vnet()) == gVnetNameToId.end())
+    if (ctxt.metadata.routing_type() == dash::route_type::RoutingType::ROUTING_TYPE_VNET &&
+        ctxt.metadata.has_vnet() && gVnetNameToId.find(ctxt.metadata.vnet()) == gVnetNameToId.end())
     {
         SWSS_LOG_INFO("Retry as vnet %s not found", ctxt.metadata.vnet().c_str());
         return false;
     }
+    if (ctxt.metadata.routing_type() == dash::route_type::RoutingType::ROUTING_TYPE_VNET_DIRECT &&
+        ctxt.metadata.has_vnet_direct() && gVnetNameToId.find(ctxt.metadata.vnet_direct().vnet()) == gVnetNameToId.end())
+    {
+        SWSS_LOG_INFO("Retry as vnet %s not found", ctxt.metadata.vnet_direct().vnet().c_str());
```

Comment:
Can you differentiate the logs for two conditions? We can just put a routing type string here.

Reply:
Updated the log to include routing type string